docs/development.rst

```diff
@@ -505,463 +505,795 @@ point, with diagrams, and an alternative workflow, see `the git docs
+object, adding the assembly name and accession, and all the relevant citations,
+with a "reason" associated with each citation:
 
----------------------
-Write parameter table
----------------------
+.. code-block:: python
 
-The parameters used in the implementation must
-also be listed in a csv file in the ``docs/parameter_tables`` directory. This ensures
-that the documentation for this model displays the parameters.
+  _genome = stdpopsim.Genome(
+      chromosomes=_chromosomes,
+      assembly_name=genome_data.data["assembly_name"],
+      assembly_accession=genome_data.data["assembly_accession"],
+      citations=[
+        stdpopsim.Citation(
+          author="Ossowski et al.",
+          year=2010,
+          doi="https://doi.org/10.1126/science.1180677",
+          reasons={stdpopsim.CiteReason.MUT_RATE},
+        ),
+        stdpopsim.Citation(
+          author="Huber et al.",
+          year=2014,
+          doi="https://doi.org/10.1093/molbev/msu247",
+          reasons={stdpopsim.CiteReason.REC_RATE},
+        ),
+        stdpopsim.Citation(
+          doi="https://doi.org/10.1093/nar/gkm965",
+          year=2007,
+          author="Swarbreck et al.",
+          reasons={stdpopsim.CiteReason.ASSEMBLY},
+        ),
+      ],
+  )
+
+The ``_species`` object contains the remaining information about the species,
+including the **effective population size** and **average generation time**,
+along with relevant citations.
+For example, the ``_species`` object for *A. thaliana* is specified as follows:
 
-Take a look at the csv files currently in ``docs/parameter_tables`` for inspiration.
-The csv file should have the format::
+.. code-block:: python
 
-    Parameter Type (units), Value, Description
+    _species = stdpopsim.Species(
+        id="AraTha",
+        ensembl_id="arabidopsis_thaliana",
+        name="Arabidopsis thaliana",
+        common_name="A. thaliana",
+        genome=_genome,
+        generation_time=1.0,
+        population_size=10**4,
+        ploidy=_species_ploidy,
+        citations=[
+            stdpopsim.Citation(
+                doi="https://doi.org/10.1890/0012-9658(2002)083[1006:GTINSO]2.0.CO;2",
+                year=2002,
+                author="Donohue",
+                reasons={stdpopsim.CiteReason.GEN_TIME},
+            ),
+            stdpopsim.Citation(
+                doi="https://doi.org/10.1016/j.cell.2016.05.063",
+                year=2016,
+                author="1001GenomesConsortium",
+                reasons={stdpopsim.CiteReason.POP_SIZE},
+            ),
+        ],
+    )
 
 
-We can check that the documentation builds properly after implementation by running
-``make`` in the docs directory and opening the Catalog page from the ``docs/_build/``
-directory. See `Documentation`_ for more details.
+Once these two objects (``_genome`` and ``_species``) are specified,
+you should be able to load and simulate the newly added species using ``stdpopsim``.
 
```

Comment:
Should the original contributor also generate tables for the docs, as done when a new demographic model is added?

Reply:
I don't think there's any tables that aren't autogenerated for species; am I forgetting something?